README.md

```diff
@@ -116,7 +116,8 @@ The following section is useful for Dockstore developers (e.g. those that want t
 The dependency environment for Dockstore is described by our 
 [Travis-CI config](https://github.com/ga4gh/dockstore/blob/develop/.travis.yml). In addition to the dependencies for 
 Dockstore users, note the setup instructions for postgres. Specifically, you will need to have postgres installed 
-and setup with the database user specified in [.travis.yml](https://github.com/ga4gh/dockstore/blob/develop/.travis.yml#L26). 
+and setup with the database user specified in [.travis.yml](https://github.com/ga4gh/dockstore/blob/develop/.travis.yml#L26).
```

Comment:
Worth putting in a note that we aim to have unit tests that run without postgres. 
Integration tests would require it